agent/command/s3_copy.go

```diff
@@ -188,43 +243,70 @@ func (c *s3copy) s3Copy(ctx context.Context,
 			s3CopyFile.Source.Path, s3CopyFile.Destination.Bucket,
 			s3CopyFile.Destination.Path)
 
-		s3CopyReq := apimodels.S3CopyRequest{
-			AwsKey:              c.AwsKey,
-			AwsSecret:           c.AwsSecret,
-			S3SourceRegion:      s3CopyFile.Source.Region,
-			S3SourceBucket:      s3CopyFile.Source.Bucket,
-			S3SourcePath:        s3CopyFile.Source.Path,
-			S3DestinationRegion: s3CopyFile.Destination.Region,
-			S3DestinationBucket: s3CopyFile.Destination.Bucket,
-			S3DestinationPath:   s3CopyFile.Destination.Path,
-			S3DisplayName:       s3CopyFile.DisplayName,
-			S3Permissions:       s3CopyFile.Permissions,
-		}
-
-		responseString, err := comm.S3Copy(ctx, td, &s3CopyReq)
+		s3CopyReq.AwsKey = s3pc.AwsKey
```

Comment:
Are we able to add this when s3CopyReq is defined?

Reply:
I moved it because I send s3copyReq to the push command, which doesn't need them, and I didn't want to send them if it's not necessary.